README.md

```diff
@@ -31,6 +31,15 @@ Also, you need to include the library as your dependency:
 libraryDependencies += "com.danielasfregola" %% "twitter4s" % "5.1"
 ```
 
+Setup using Giter8
+------------------
+
+If you are starting from scratch, you can use [`twitter4s.g8`](https://github.com/DanielaSfregola/twitter4s.g8) template to start your project. This template contains examples for both REST and streaming client.
```

Comment:
Please change "streaming" to "Streaming" for consistency with the rest of the file

Reply:
Totally agree